src/FlagContext.ts

```diff
@@ -1,5 +1,24 @@
 import React from 'react';
+import type { IContext, IVariant, UnleashClient } from 'unleash-proxy-client';
 
-const FlagContext = React.createContext(null);
+type eventArgs = [Function, any];
+
+export interface IFlagContextValue {
+  on(event: string, ...args: eventArgs): UnleashClient;
```

Comment:
`eventArgs` can be inline (not exported type), and I think third argument is optional, isn't it?